zad3/app/controllers/bestiary_controller.rb

```diff
@@ -1,14 +1,30 @@
 # frozen_string_literal: true
 
 class BestiaryController < ApplicationController
-  def index
 
-  end
 
-  def novigrad_monsters
 
+  def index
+    begin
+    @monster = Array.new
+ 
+      monsters.each {|f|  @monster.push(Potwor.new(f))}
+      @monster.each {|f| f.save}
+    
+    rescue
+    @monster =Potwor.all
+    end
```

Comment:
Jedyne czym powinna się zajmować ta metoda to przypisanie odpowiedniej kolekcji potworów do zmiennej instancyjnej `@monsters`, po to żeby mieć do niej dostęp w widoku 😉 Nie trzeba ich pushować jeden po drugim i dodatkowo wiemy z treści zadania, że metoda `monsters` zwraca już odpowiednio spreparowaną kolekcję. Tak więc jedyne co tutaj trzeba napisać to:
```ruby
@monsters = monsters
```